content/influxdb/cloud-dedicated/query-data/influxql/basic-query.md

```diff
@@ -0,0 +1,215 @@
+Use `now()` to return the timestamp for the current time (UTC).
+
+##### Query all data from the last month
+
+```sql
+SELECT * FROM home WHERE time >= now() - INTERVAL '1 month'
+```
+
+##### Query one day of data data from a week ago
+```sql
+SELECT *
+FROM home
+WHERE
+  time >= now() - INTERVAL '7 days'
+  AND time <= now() - INTERVAL '6 days'
+```
+{{% /expand %}}
+
+{{% expand "Query with absolute time boundaries" %}}
+
+To query data from absolute time boundaries, compare the value of the `time` column
+to a timestamp literal.
+Use the `AND` logical operator to chain together multiple predicates and define
+both start and stop boundaries for the query.
+
+{{% influxdb/custom-timestamps %}}
+```sql
+SELECT
+  *
+FROM
+  home
+WHERE
+  time >= '2022-01-01T08:00:00Z'
+  AND time <= '2022-01-01T20:00:00Z'
+```
+{{% /influxdb/custom-timestamps %}}
+
+{{% /expand %}}
+{{< /expand-wrapper >}}
+
+### Query data without time boundaries
+
+To query data without time boundaries, do not include any time-based predicates
+in your `WHERE` clause.
+If a time range is not defined in the `WHERE` clause, the default time range is
+the Unix epoch (`1970-01-01T00:00:00Z`) to _now_.
+
+{{% warn %}}
+Querying data _without time bounds_ can return an unexpected amount of data.
+The query may take a long time to complete and results may be truncated.
+{{% /warn %}}
+
+```sql
+SELECT * FROM home
+```
+
+### Query specific fields and tags
+
+To query specific fields, include them in the `SELECT` clause.
+If querying multiple fields or tags, comma-delimit each.
+If a field or tag key includes special characters or spaces or is case-sensitive,
+wrap the key in _double-quotes_.
+
+```sql
+SELECT time, room, temp, hum FROM home
+```
+
+### Query fields based on tag values
+
+- In the `SELECT` clause, include fields you want to query and tags you want to base conditions on.
+- In the `WHERE` clause, include predicates that compare the tag identifier to a string literal.
+  Use [logical operators](/influxdb/cloud-dedicated/reference/influxql/where/#logical-operators) to chain multiple predicates together and apply
+  multiple conditions.
+
+```sql
+SELECT * FROM home WHERE room = 'Kitchen'
+```
+
+### Query points based on field values
+
+- In the `SELECT` clause, include fields you want to query.
+- In the `WHERE` clause, include predicates that compare the field identifier to a value or expression.
+  Use [logical operators](/influxdb/cloud-dedicated/reference/influxql/where/#logical-operators) (`AND`, `OR`) to chain multiple predicates together
+  and apply multiple conditions.
+
+```sql
+SELECT co, time FROM home WHERE co >= 10 OR co <= -10
+```
+
+### Alias queried fields and tags
+
+To alias or rename fields and tags that you query, use the `AS` clause.
+After the tag, field, or expression you want to alias, pass `AS` followed by the alias name as a string literal in double quotes (`"`)--for example:
```

Comment:
SQL and InfluxQL string literals use single quotes. Double quotes are used to quote identifiers with special characters or spaces.

```suggestion
After the tag, field, or expression you want to alias, pass `AS` followed by the alias name as an identifier (wrap in double quotes (`"`) if the alias includes spaces or special characters)--for example:
```

Reply:
I'm pretty sure aliases only worked when double quoted using `influx3`. Could also be the client. I'll test again.